2.evaluate_models/classify_genotypes/genotype_classification_evaluation.ipynb

```diff
@@ -104,18 +104,18 @@
    "execution_count": 3,
    "metadata": {
     "execution": {
-     "iopub.execute_input": "2024-06-24T21:00:48.528613Z",
-     "iopub.status.busy": "2024-06-24T21:00:48.528502Z",
-     "iopub.status.idle": "2024-06-24T21:00:48.573776Z",
-     "shell.execute_reply": "2024-06-24T21:00:48.573330Z"
+     "iopub.execute_input": "2024-07-22T18:13:21.505670Z",
+     "iopub.status.busy": "2024-07-22T18:13:21.505419Z",
+     "iopub.status.idle": "2024-07-22T18:13:23.506858Z",
+     "shell.execute_reply": "2024-07-22T18:13:23.506434Z"
     },
     "jukit_cell_id": "zFHiLC64Ss"
    },
    "outputs": [],
    "source": [
     "data_path = pathlib.Path(f\"{root_dir}/1.train_models/classify_genotypes/data\")\n",
     "\n",
-    "evaldf = pd.read_parquet(f\"{data_path}/nf1_final_model_eval_data.parquet\")\n",
+    "evaldf = pd.read_parquet(f\"{data_path}/nf1_model_pre_evaluation_results.parquet\")\n",
```

Comment:
Just curious, why is it called "pre" evaluation results and not just evaluation?

Reply:
This is to distinguish between the evaluation results, and the data to be used for creating those results